web-server/src/components/Service/SystemStatus.tsx

```diff
@@ -0,0 +1,174 @@
+import { Box, CircularProgress, Divider, useTheme } from '@mui/material';
+import { alpha } from '@mui/material/styles';
+import { FC, useEffect } from 'react';
+
+import { ServiceNames } from '@/constants/service';
+import { StreamEventType } from '@/constants/stream';
+import { CardRoot } from '@/content/DoraMetrics/DoraCards/sharedComponents';
+import { serviceSlice } from '@/slices/service';
+import { useDispatch, useSelector } from '@/store';
+
+import { FlexBox } from '../FlexBox';
+import { useOverlayPage } from '../OverlayPageContext';
+import { Line } from '../Text';
+
+const serviceTitle: Record<ServiceNames, string> = {
+  [ServiceNames.API_SERVER]: 'Backend Server',
+  [ServiceNames.REDIS]: 'Redis Database',
+  [ServiceNames.POSTGRES]: 'Postgres Database',
+  [ServiceNames.SYNC_SERVER]: 'Sync Server'
+};
+
+const serviceColor: Record<ServiceNames, string> = {
+  [ServiceNames.API_SERVER]: '#06d6a0',
+  [ServiceNames.REDIS]: '#ef476f',
+  [ServiceNames.POSTGRES]: '#ff70a6',
+  [ServiceNames.SYNC_SERVER]: '#ab34eb'
+};
+
+export const SystemStatus: FC = () => {
+  const dispatch = useDispatch();
+  const theme = useTheme();
+  const loading = useSelector((state) => state.service.loading);
+  const services = useSelector((state) => state.service.services);
+
+  useEffect(() => {
+    const eventSource = new EventSource(`/api/stream`);
+
+    eventSource.onmessage = (event) => {
+      const data = JSON.parse(event.data);
+
+      if (data.type === StreamEventType.StatusUpdate) {
+        dispatch(serviceSlice.actions.setStatus({ statuses: data.statuses }));
+      }
+      if (data.type === StreamEventType.LogUpdate) {
+        const { serviceName, content } = data;
+        const newLines = content.split('\n');
+        const trimmedLines = newLines.filter(
+          (line: string) => line.trim() !== ''
+        );
+        dispatch(
+          serviceSlice.actions.setServiceLogs({
+            serviceName,
+            serviceLog: trimmedLines
+          })
+        );
+      }
+    };
+
+    return () => {
+      eventSource.close();
+      dispatch(serviceSlice.actions.resetState());
+    };
+  }, [dispatch]);
+
+  const { addPage } = useOverlayPage();
+
+  const handleCardClick = (serviceName: ServiceNames) => {
+    addPage({
+      page: {
+        ui: 'system_logs',
+        title: `${serviceTitle[serviceName]} Logs`,
+        props: { serviceName }
+      }
+    });
+  };
+
+  return (
+    <FlexBox col gap={2} sx={{ padding: '16px' }}>
+      <Line bold white fontSize="24px" sx={{ mb: 2 }}>
+        System Status
+      </Line>
+      <Divider sx={{ mb: 2, backgroundColor: 'rgba(255, 255, 255, 0.2)' }} />
+
+      {loading ? (
+        <Box
+          sx={{
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'center'
+          }}
+        >
+          <CircularProgress color="primary" size={50} />
+        </Box>
```

Comment:
use `<Flexbox alignCenter justifyCenter/> instes of `Box`

Reply:
or `<Flexbox centered/> `